src/antelope/wallets/authenticators/WalletConnectAuth.ts

```diff
@@ -55,8 +56,9 @@ export class WalletConnectAuth extends EVMAuthenticator {
             this.clearAuthenticator();
             const address = getAccount().address as addressString;
             try {
-                await super.login(network);
-            } catch (e) {
+                const authenticator = useAccountStore().loggedAccount.authenticator as EVMAuthenticator;
+                await useAccountStore().loginEVM({ authenticator,  network });
+            }catch(e){
```

Comment:
This may cause an infinite loop since loginEVM method calls the authenticator login again. 
Now I'm thinking... what is your intention? 

https://github.com/telosnetwork/telos-wallet/blob/0b2eee1a18640db67d692f43d328dd0d5c96b7b3/src/antelope/stores/account.ts#L161